moderator/settings.py

```diff
@@ -248,3 +248,24 @@ def _username_algo(email):
 
 # Enable dev login
 ENABLE_DEV_LOGIN = config("ENABLE_DEV_LOGIN", default=False, cast=bool)
+FROM_NOREPLY = config(
+    "FROM_NOREPLY",
+    default="No Reply - Mozilla Moderator <no-reply@moderator-mozilla.org",
+)
+
+if DEV and DEBUG:
+    EMAIL_LOGGING_REAL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
+    EMAIL_HOST = "mailcatcher"
+    EMAIL_HOST_USER = ""
+    EMAIL_HOST_PASSWORD = ""
+    EMAIL_PORT = 1025
+    EMAIL_USE_TLS = False
+else:
+    # AWS SES configuration
+    EMAIL_BACKEND = "django_ses.SESBackend"
+    AWS_ACCESS_KEY_ID = config("AWS_ACCESS_KEY_ID")
+    AWS_SECRET_ACCESS_KEY = config("AWS_SECRET_ACCESS_KEY")
+    AWS_REGION_NAME = config("AWS_DEFAULT_REGION", default="us-west-2")
+    AWS_REGION_ENDPOINT = config(
```

Comment:
https://github.com/django-ses/django-ses#full-list-of-settings seems to suggest that these should be `AWS_SES_REGION_NAME` and `AWS_SES_REGION_ENDPOINT`.

Reply:
aiui, I think those AWS_SES_REGION_* envvars are for when you don't want to use the same AWS creds for multiple AWS service API calls. I leave that decision to yall - as I understand it, the only AWS API called here is SES (everything else, like the RDS db, is connected to through other mechanisms).